Source/Python/WarpXWrappers.h

```diff
@@ -85,13 +88,27 @@ extern "C" {
 
     amrex::Real warpx_getProbHi(int dir);
 
-    long warpx_getNumParticles(int speciesnumber);
+    long warpx_getNumParticles(const char* char_species_name);
+
+    amrex::ParticleReal** warpx_getParticleStructs(
+        const char* char_species_name, int lev, int* num_tiles,
+        int** particles_per_tile);
+
+    amrex::ParticleReal** warpx_getParticleArrays(
+        const char* char_species_name, const char* char_comp_name, int lev,
+        int* num_tiles, int** particles_per_tile);
+
+    int warpx_getParticleCompIndex(
+        const char* char_species_name, const char* char_comp_name);
+
+    void warpx_addRealComp(
+        const char* char_species_name, const char* char_comp_name, bool comm);
 
-    amrex::ParticleReal** warpx_getParticleStructs(int speciesnumber, int lev,
-                                                   int* num_tiles, int** particles_per_tile);
+    int warpx_getParticleCompIndex(
+        const char* char_species_name, const char* char_comp_name);
 
-    amrex::ParticleReal** warpx_getParticleArrays(int speciesnumber, int comp, int lev,
-                                                  int* num_tiles, int** particles_per_tile);
+    void warpx_addRealComp(
+        const char* char_species_name, const char* char_comp_name, bool comm);
```

Comment:
These seem to be repeated definitions of the functions on lines 101 and 104.

Reply:
Nice catch. Thanks!